deployment/modules/cloudflare/backend/locals.tf

```diff
@@ -8,6 +8,5 @@ locals {
   domain_prefix = "${local.domain_stage}${local.domain_env}"
   domain        = "data.${local.domain_prefix}immich.app"
 
-  monitoring_url   = var.env == "prod" ? "https://cf-workers.monitoring.immich.cloud" : "https://cf-workers.monitoring.dev.immich.cloud"
-  monitoring_token = var.vmetrics_api_token
+  vmetrics_data_api_url = var.env == "prod" ? "https://data.immich.cloud" : "https://data.immich.cloud"
```

Comment:
I assume this should be something else for dev

Reply:
I'll be fixing this when I add the github actions part